packages/cmcd-validator-library/src/tests/index.js

```diff
@@ -1,4 +1,4 @@
-import { testCMCDQueryValidator, testCMCDHeaderValidator, testCMCDJsonValidator } from './CMCDValidator/index.js';
```

Comment:
Is this deletion expected?

Reply:
Yes, it's expected because testCMCDHeaderValidator and testCMCDJsonValidator do not exist anymore.